src/worker/media/media_payload_writer.py

```diff
@@ -0,0 +1,38 @@
+from core.feluda import Feluda
+from core.logger import Logger
+from core.queue.amazon_mq import AmazonMQ
+from time import sleep
+import uuid
+import sys
+
+log = Logger(__name__)
+
+try:
+    feluda = Feluda("worker/media/config.yml")
+    feluda.setup()
+    queue_config = feluda.config.amazon_queue
+    media_index_queue = queue_config.parameters.queues[0]["name"]
+    amazom_queue_manager = AmazonMQ(queue_config)
```

Comment:
the api should be
```
feluda = Feluda("worker/media/config.yml")
feluda.setup()
queue = Feluda.queue
```

We should not expose internal details like which queue feluda is using to the worker.